sdk/src/impl/whirlpool-impl.ts

```diff
@@ -522,10 +550,20 @@ export class WhirlpoolImpl implements Whirlpool {
 
     txBuilder.addInstruction(positionIx);
 
+    if (createAndInitWsolAccountIx) {
+      // Append the wSOL token account creation IXs to the end of the first TX
+      tokenAccountsTxBuilder.addInstruction(createAndInitWsolAccountIx);
+    }
+
+    if (unwrapWsolIx) {
+      // Append the wSOL unwrap IX (i.e. wSOL token account close IX) to the end of the second TX
+      txBuilder.addInstruction(unwrapWsolIx);
```

Comment:
can we append the wSOL wrapping ix (create & cleanup) to this txBuilder?
We seem to be adding a lot more paths to keep track of the wSOL ixs.